docs/mass_mapping.md

```diff
@@ -0,0 +1,111 @@
+# Reproducing our mass-mapping results
+
+This .md file has all the basic information required to reproduce the results from the [MMGAN paper](https://arxiv.org/abs/2410.24197).
+
+## Setup Instructions
+Install the required modules via anaconda:
+```
+conda create --name <env> --file requirements.txt
+```
+Note that you may need to change the PyTorch version, depending on your CUDA distribution.
+
+## Update Config
+Update ```configs/mass_map.yml``` with the path to your data, where you want to store checkpoints, and with the path to additional data such as masks.
+
+## Data Setup
```

Comment:
I think we need to add more detail to the data preparation description. It should be somehow clearer what steps/scripts/notebooks to use and in which order. It'd be good to link to the dataset sources we used, e.g. kappaTNG sims, and the COSMOS dataset.

Reply:
@tobias-liaudat ready for you to look at again, I've updated the data setup section